src/app/pages/staging/index.js

```diff
@@ -436,7 +436,7 @@ const Staging = () => {
 	 * 
 	 * @param {string} type One of 'all', 'files', or 'db'
 	 */
-	const deployStaging = ( type ) => {
+	const deplonfdaging = ( type ) => {
```

Comment:
This will need to be reverted

Reply:
Thanks for spotting this. It was at least changed in both places so nothing broke, but reverting is the best so it is clear what the method is!